--- Makefile.in	2007-12-21 15:13:41.000000000 +0100
+++ Makefile.in.oden	2008-10-08 16:28:50.000000000 +0200
@@ -7,6 +7,7 @@
 MANDIR		= @mandir@
 INSTALL		= @INSTALL@
 MKINSTALLDIRS	= @srcdir@/mkinstalldirs
+DESTDIR         =
 
 LIBS		= -lpcap -lnet
 
@@ -19,18 +20,18 @@
 	${CC} -o dhcptool ${OBJS} ${LDFLAGS} ${LIBS}
 
 install-bin: dhcptool
-	${MKINSTALLDIRS} ${BINDIR}
-	${INSTALL} dhcptool ${BINDIR}
+	${MKINSTALLDIRS} ${DESTDIR}${BINDIR}
+	${INSTALL} dhcptool ${DESTDIR}${BINDIR}
 
 install-man:
-	${MKINSTALLDIRS} ${MANDIR}/man1
-	${INSTALL} dhcptool.1 ${MANDIR}/man1
+	${MKINSTALLDIRS} ${DESTDIR}${MANDIR}/man1
+	${INSTALL} dhcptool.1 ${DESTDIR}${MANDIR}/man1
 
 install: install-bin install-man
 
 uninstall:
-	rm -f ${BINDIR}/dhcptool
-	rm -f ${MANDIR}/man1/dhcptool.1*
+	rm -f ${DESTDIR}${BINDIR}/dhcptool
+	rm -f ${DESTDIR}${MANDIR}/man1/dhcptool.1*
 
 clean:
 	rm -f dhcptool *.o *.core
